astar_planner:
  ros__parameters:
    planner:
      type: "AStar"
      theta_resolution: 8.0
      xy_tolerance: 0.1
      theta_tolerance: 0.1
      timeout: 20.0

    map:
      width: 20.0
      height: 20.0
      resolution: 0.1
